t/plugin/opentelemetry.t

```diff
@@ -95,113 +77,34 @@ __DATA__
             if code >= 300 then
                 ngx.status = code
             end
-            ngx.say(body)
+            ngx.say("done")
```

Comment:
Our previous tests were all body, why should we change to done. Can we keep the code style consistent?

Reply:
Okay